docs/index.md

```diff
@@ -1,40 +1,38 @@
 ---
-home: true
-heroImage: /logo.svg
-actionText: Get Started
-actionLink: /guide/
-
-altActionText: Learn More
-altActionLink: /guide/why
+layout: home
+
+hero:
+  name: Vite
+  text: The Next Generation Frontend Tooling
+  tagline: Get ready for the brazing fast build environment that can finally catch up with you.
```

Comment:
We're going to love getting these PRs, the design is awesome ❤️
 
For the text, I think it sounds better without the initial "The".
And I think it is "blazing", but maybe the word is a bit overused. I like the gist of the tagline though. What if let the user do the math about it being fast and just say the can finally catch up with you line? 
```suggestion
  text: Next Generation Frontend Tooling
  tagline: Get ready for a development environment that can finally catch up with you.
```

Reply:
Thanks! Great idea, the reason why added "The" was more of a design reason where line break happens at weird place 😅

<img width="1392" alt="Screen Shot 2022-05-22 at 0 44 06" src="https://user-images.githubusercontent.com/3753672/169659343-75fa820d-8af4-4dec-a737-3bfcbf6b576c.png">

So I'll apply the both changes, and then add some custom css to adjust it!

Reply:
And now, oh yeah.

<img width="1392" alt="Screen Shot 2022-05-22 at 0 48 16" src="https://user-images.githubusercontent.com/3753672/169659519-fce0dcdc-bc0c-42d4-b63b-048797375c88.png">

Reply:
Lovely!